services/driver-service/src/test/java/com/Harevich/driverservice/constants/TestConstants.java

```diff
@@ -0,0 +1,43 @@
+package com.Harevich.driverservice.constants;
+
+public class TestConstants {
```

Comment:
Утилитные классы/классы с константами обычно делают не наследуемыми и с приватным дефолтным конструктором, чтобы не было возможности создать экземпляр класса.
```suggestion
@NoArgsConstructor(access = AccessLevel.PRIVATE)
public final class TestConstants {
```